### DISCUSSION TITLE
Title: PushGateway Installation on Windows 2008 Server

### DISCUSSION THREAD

**Discussion by narayanalak** [prometheus/pushgateway#662] [Q&A]
Subject: I'm trying to install pushgateway on windows 2008 server and the service is going to "paused" state.
pls find the attachment for errors:
How to fix this issue
![WhatsApp Image 2024-06-27 at 9 59 36 AM](https://github.com/prometheus/pushgateway/assets/173983902/1a868fb5-f00a-4f0d-9862-956af3bcb7ec)
![WhatsApp Image 2024-06-27 at 9 58 48 AM](https://github.com/prometheus/pushgateway/assets/173983902/20f484d2-fdd1-44de-b346-029deb4f9461)

**Comment by beorn7** [comment]
I moved this to discussions, as this seems more an MS Windows config problem. (Happy to make it an issue again should there be evidence that this is a bug in the Pushgateway.)

Maybe you'll have better chances for an answer if you ask in MS Windows specific support forums for help.
